internal/pkg/auth/sso/auth0_utils.go

```diff
@@ -4,16 +4,19 @@ import (
 	"net/url"
 	"strings"
 
+	"github.com/confluentinc/cli/internal/pkg/types"
 	testserver "github.com/confluentinc/cli/test/test-server"
 )
 
 var auth0ClientIds = map[string]string{
-	"cpd":              "7rG4pmRbnMn5mIsEBLAP941IE1x2rNqC",
-	"devel":            "sPhOuMMVRSFFR7HfB606KLxf1RAU4SXg",
-	"fedramp-internal": "0oa7c9gkc6bHBD2OW1d7",
-	"prod":             "oX2nvSKl5jvBKVgwehZfvR4K8RhsZIEs",
-	"stag":             "8RxQmZEYtEDah4MTIIzl4hGGeFwdJS6w",
-	"test":             "00000000000000000000000000000000",
+	"cpd":          "7rG4pmRbnMn5mIsEBLAP941IE1x2rNqC",
+	"devel":        "sPhOuMMVRSFFR7HfB606KLxf1RAU4SXg",
+	"infra-us-gov": "0oa7c9gkc6bHBD2OW1d7",
+	"prod-us-gov":  "TODO",
+	"devel-us-gov": "TODO",
```

Comment:
@xuhui-lu, can you look these up for me?

Reply:
prod-us-gov: 0oa41ih4ms3TVVAT04h7
We have not provisioned the devel-us-gov tenant, so I think let's leave it as empty for now.